omod/src/main/webapp/resources/htmlforms/Covid19-ReferralForm.xml

```diff
@@ -0,0 +1,1252 @@
+                                                    </div>
+                                                    <div class="col-md-4">
+                                                        <label>Current</label>
+                                                        <obs conceptId="159395" class="col-md-3" />
+                                                    </div>
+                                                </div>
+                                            </div>
+                                            <div class="col-md-6">
+                                                <label>Blood Pressure(BP)</label>
+                                                <div class="row">
+                                                    <div class="col-md-4">
+                                                        <label>Arrival</label>
+                                                        <obs id="arrival" conceptId="159395" answerLabel="arrival" class="col-md-3" />
+                                                    </div>
+                                                    <div class="col-md-4">
+                                                        <label>Current</label>
+                                                        <obs conceptId="159395" class="col-md-3" />
+                                                    </div>
+                                                </div>
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+
+                                <div class="card">
+                                    <div class="card-body">
+                                        <div class="row">
+                                            <div class="col-md-6">
+                                                <label>Blood Glucose(RBG)</label>
+                                                <div class="row">
+                                                    <div class="col-md-4">
+                                                        <label>Arrival</label>
+                                                        <obs id="arrival" conceptId="159395" answerLabel="arrival" class="col-md-3" />
+                                                    </div>
+                                                    <div class="col-md-4">
+                                                        <label>Current</label>
+                                                        <obs conceptId="159395" class="col-md-3" />
+                                                    </div>
+                                                </div>
+                                            </div>
+                                            <div class="col-md-6">
+                                                <label>AVPU(Alert, to Pain, to 
```

Comment:
use this concept (166410) for AVPU and present answers as drop down, remove them from brackets in the label